ckanext/stcndm/schemas/geo_codeset.yaml

```diff
@@ -0,0 +1,39 @@
+scheming_version: 1
+about: 'Statistics Canada Dissemination: Geo Codes'
+about_url: http://github.com/open-data/ckanext-stcndm
+catalog_type_label:
+  en: Geo Codeset
+  fr: Geo Codeset
+dataset_type: geo_codeset
+form_languages:
+- en
+- fr
+languages_label:
+- eng; CAN
+- fra; CAN
+
+dataset_fields:
+
+- field_name: id
+  label:
+    en: Internal Unique ID
+    fr: ID interne unique
+
+- field_name: value
+  label:
+    en: Code value
+    fr: Valeur du code
+  preset: fluent_text
+
+- field_name : label
+  label:
+    en: Descriptive label
+    fr: Étiquette descriptive
+  preset: fluent_text
+
+resource_fields:
+# XXX a quick hack to allow the creation of one resource to make the
+# package controller happy. a better fix would be to disable the resource
+# edit page by overriding the controller
+- field_name: url
+  required: true
```

Comment:
note that the resource_fields hack is no longer required. Remove all your resource_fields and use the latest scheming to make resourceless datasets